client/src/main/scala/org/apache/celeborn/client/ClientUtils.scala

```diff
@@ -37,4 +39,20 @@ object ClientUtils {
     }
     true
   }
+
+  /**
+   * If startMapIndex > endMapIndex, means partition is skew partition.
+   * locations will split to sub-partitions with startMapIndex size.
+   *
+   * @param conf cleborn conf
+   * @param startMapIndex shuffle start map index
+   * @param endMapIndex shuffle end map index
+   * @return true if read skew partition without map range
+   */
+  def readSkewPartitionWithoutMapRange(
+      conf: CelebornConf,
+      startMapIndex: Int,
+      endMapIndex: Int): Boolean = {
+    conf.clientAdaptiveOptimizeSkewedPartitionReadEnabled && startMapIndex > endMapIndex
```

Comment:
QQ: Can `startMapIndex > endMapIndex` when `!clientAdaptiveOptimizeSkewedPartitionReadEnabled` ?

Reply:
It can't be.

Reply:
I think we need some comment to explain this.